languages/rust/exercises/concept/structs/.docs/hints.md

```diff
@@ -0,0 +1,16 @@
+## General
+- [https://learning-rust.github.io/docs/b2.structs.html#C-like-structs](https://learning-rust.github.io/docs/b2.structs.html#C-like-structs) provides a good overview of the different types of structs and their syntax.
+
+- [https://doc.rust-lang.org/book/ch05-01-defining-structs.html](https://doc.rust-lang.org/book/ch05-01-defining-structs.html) in _The Rust Programming Language_ book talks about how to define structs.
+
+- [https://doc.rust-lang.org/book/ch05-03-method-syntax.html](https://doc.rust-lang.org/book/ch05-03-method-syntax.html) in _The Rust Programming Language_ showcases the syntax for defining methods on structs.
+
+- Notice that the `get_name` method returns a `&str` when the `name` field on the `User` struct is a `String`. How can we get `&str` and `String` to play nice with each other?
```

Comment:
This is an excellent hint, where the solution isn't spelled out! You could consider adding a link to an external resource that can help the student finding the answer.